plugins/modules/files/archive.py

```diff
@@ -66,13 +66,14 @@
     type: bool
     default: false
 notes:
-    - Requires tarfile, zipfile, gzip and bzip2 packages on target host.
-    - Requires lzma or backports.lzma if using xz format.
-    - Can produce I(gzip), I(bzip2), I(lzma) and I(zip) compressed files or archives.
+    - Can produce I(gzip), I(bzip2), I(lzma), and I(zip) compressed files or archives.
+    - Requires C(tarfile), C(zipfile), C(gzip), and C(bz2) packages on target host. These are all part of the Python standard library for Python 2 and 3.
```

Comment:
> I once heard that there are some distributions which remove parts of the stdlib (resp. require extra packages to be installed for them).

That sounds horrifying.

> I guess it might suffice to keep that list in the `Notes` section though. WDYT? (That statement used to be there since the very first initial commit ([ansible/ansible-modules-extras#2323](https://github.com/ansible/ansible-modules-extras/pull/2323)).)

I am inclined not to include it; we don't list any other python stdlib modules as a dependencies. If you insist on keeping it, something like the following that doesn't list it as required would be better.

```suggestion
    - This module uses C(tarfile), C(zipfile), C(gzip), and C(bz2) packages on the target host to create archives. These are part of the Python standard library for Python 2 and 3.
```

Reply:
> > I once heard that there are some distributions which remove parts of the stdlib (resp. require extra packages to be installed for them).
> 
> That sounds horrifying.

It's unfortunately not that uncommon, for example Debian doesn't have `venv` in the default Python 3 installation, you need to install it separately (or install [python3-full](https://packages.debian.org/bullseye/python3-full) to really get everything).

Reply:
Oops, it looks like my suggestion needed to be line wrapped:

```
01:52 ERROR: Found 1 pep8 issue(s) which need to be resolved:
01:52 ERROR: plugins/modules/files/archive.py:70:161: E501: line too long (180 > 160 characters) (100%)
01:52 See documentation for help: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/pep8.html
```